e-store/e-store/Router/StoreRouter.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  StoreRouter.swift
+//  e-store
+//
+//  Created by Ahmed Atef on 09/04/2023.
+//
+
+import Foundation
+
+protocol StoreRouterProtocol {
+    //TODO: - Navigation function here
+}
+
+class StoreRouter {
+    //TODO: - Implement navigation function here
```

Comment:
where the implementation

Reply:
who want to navigate will implement its navigation function